tests/test_util.ts

```diff
@@ -33,6 +38,9 @@ export class TestSuite {
           await f();
         }
         await test.func();
```

Comment:
Could you move this cleanup process to the finally block? If the test fails, functions registered by `afterEach` will be skipped.
```suggestion
        try {
          await test.func();
        } finally {
          for (const f of this.afterEachs) {
            await f();
          }
        }
```

Reply:
Ah, sorry about that. That is more than a small nitpick 😄 Fixed. Thanks!